content/patterns/button/examples/button_idl.html

```diff
@@ -168,24 +168,24 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>a</code></td>
               <td>
                 <ul>
-                  <li>Set in javascript with <code>button.ariaPressed = 'true';</code>.</li>
-                  <li>Identifies the button as a toggle button.</li>
-                  <li>Indicates the toggle button is pressed.</li>
+                  <li>JavaScript で <code>button.ariaPressed = 'true';</code> を使用して設定します。</li>
+                  <li>ボタンをトグルボタンとして識別します。</li>
+                  <li>トグルボタンが押されていることを示します。</li>
                 </ul>
               </td>
             </tr>
             <tr data-test-id="svg-aria-hidden">
               <td></td>
               <th scope="row"><code>aria-hidden="true"</code></th>
               <td><code>svg</code></td>
-              <td>Excludes SVG from accessible name calculation for the button.</td>
+              <td>ボタンのアクセシブルネーム計算から SVG を除外します。</td>
```

Comment:
accessible name を「アクセシブルな名前」としてみました。また別の場所で「デフォルトでは、アクセシブルネームはボタン要素内のテキストコンテンツから算出されます。」と訳されているのに合わせて、calculation を「算出」にしてみました。
```suggestion
              <td>ボタンのアクセシブルな名前の算出から、SVG を除外します。</td>
```